packages/crypto/src/constants.ts

```diff
@@ -21,3 +21,5 @@ export const IV_INFO = new Uint8Array([
   41, 180, 122, 198, 166, 48, 185, 46, 196, 207, 125, 35, 69, 8, 208, 175, 151,
   113, 201, 158, 80,
 ]); //base_nonce
+export const SIGNER_PUBLIC_KEY =
```

Comment:
Nit: let's be explicit about the fact that we're talking about Turnkey's signer, and in prod? `TURNKEY_PRODUCTION_SIGNER_ENCLAVE_PUBLIC_KEY` would be the explicit const name, but I'd settle for `PRODUCTION_SIGNER_PUBLIC_KEY` given it's in the Turnkey SDK already, and used in "verify enclave signature" functionality.